website/docs/docs/intro.md

```diff
@@ -18,4 +19,20 @@ projects, or you can build your own version of `sandpack`, on top of our
 standard components and utilities. As you walk through this guide, you will get
 deeper into our ecosystem.
 
-<NestedSandpack />
+### Getting Started
+
+<div class="intro-section">
+  <IntroCard title="Install" description="Install Sandpack package and start coding in minutes." href="/getting-started/install" actionText="Access ->" />
```

Comment:
@tromika could you please review these copies? Can we make the "Install" one even longer, in order to keep all cards in two lines of text?

Reply:
@danilowoz yeah added a new copy!